features/Clapi.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Clapi
+  As a Centreon admin
+  I want to configure my centreon by command line
+  To industrialize it
+
+  Background:
+    Given I am logged in a Centreon server
+
+  @critical
+  Scenario: import/export
+    Given a configuration
```

Comment:
In your scenario you cannot have two When steps. There is only one event that leads to the expected outcome. So I would rephrase it as such (and edit context methods accordingly).

Given a Clapi configuration file
And it was imported
When I export the configuration through Clapi
Then the exported file is similar to the imported filed